config/versions/v5_test.go

```diff
@@ -0,0 +1,67 @@
+package versions
+
+import (
+	"bytes"
+	"context"
+	"encoding/json" //nolint:depguard // Direct use of golang json for Compact func
+	"strings"
+	"testing"
+
+	"github.com/buger/jsonparser"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	rOHLT    = `"respectOrderHistoryLimits":true,`
```

Comment:
```suggestion
```
This is unused

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/9379a9df06